socketio/src/asynchronous/socket.rs

```diff
@@ -115,10 +115,34 @@ impl Socket {
                 Some(vec![bin_data]),
             )),
             Payload::String(str_data) => {
+                // is there any reason this is different than the blocking client ?
                 serde_json::from_str::<IgnoredAny>(&str_data)?;
```

Comment:
Good point! Could you correct it please? Should be the same as the synchronous client.

Reply:
Perhaps move to a pub(crate) fn on Packet?